src/stacks/baseNestedStack.ts

```diff
@@ -33,30 +33,48 @@ export default class BaseNestedStack extends cfn.NestedStack {
   protected setupAlarm(
     localName: string,
     metricName: string,
-    localConf: config.ConfigMetricAlarms,
+    localConf: config.ConfigMetricAlarm,
     dimensions?: object,
   ): void {
-    const autoResolve = config.configAutoResolve(this.defaultType, metricName, localConf);
-    const isEnabled = config.configIsEnabled(this.defaultType, metricName, localConf);
-    const suffix = config.configAlarmSuffix(this.defaultType, metricName, localConf);
-    const fullMetricName = suffix ? `${metricName}-${suffix}` : metricName;
+    const isEnabled = localConf.enabled !== false;
```

Comment:
Isn't `isEnabled` redundant?

Reply:
It's being used only once in L41 and yet it's being checked for falsy value.

Reply:
We want to make sure that undefined is not false as by default it should be enabled

Reply:
This allows you to disable alerts for all topics or just single topic while still keeping all configs

Reply:
What I meant is that it could be inlined since it's used only once

Reply:
But I'll leave it to you, approving :slightly_smiling_face: